akka-docs/src/test/java/jdocs/stream/operators/converters/ToFromJavaIOStreams.java

```diff
@@ -80,4 +86,41 @@ public void demonstrateFromJavaIOStreams()
         "SOME RANDOM INPUT",
         new String(((ByteArrayOutputStream) outputStream).toByteArray(), charset));
   }
+
+  @Test
+  public void demonstrateAsJavaInputStream() throws Exception {
+
+    // #asJavaInputStream
+    final Sink<ByteString, InputStream> sink = StreamConverters.asInputStream();
+    final List<ByteString> list =
+        Collections.singletonList(ByteString.fromString("Some random input"));
```

Comment:
No need for this intermediate `List`,  use `Source.single` just like in the Scala sample.

Reply:
Done